exonum-java-binding/tutorials/car-registry/car-registry-messages/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.example.car</groupId>
+    <artifactId>car-registry-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>car-registry-messages</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Exonum Service Messages</name>
+
+  <properties>
+    <!--
+    Use clientRelease for messages, as arbitrary clients may need the messages.
+    -->
+    <java.compiler.release>${java.compiler.clientRelease}</java.compiler.release>
+    <os-maven-plugin.version>1.6.2</os-maven-plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+    </dependency>
+
+    <!--
+    TODO: Shall we include them, if by default Exonum types are not really needed
```

Comment:
it may need for advanced users and I believe they can add the dependency themselves.